src/IIS/Manager/Base/BaseSiteManager.cs

```diff
@@ -1,624 +1,680 @@
+                        .Delete(site.ApplicationDefaults.ApplicationPoolName);
 
-                        exists = false;
-                    }
-                    else
-                    {
-                        exists = true;
-                        return site;
-                    }
+                    exists = false;
                 }
                 else
                 {
-                    exists = false;
+                    exists = true;
+                    return site;
                 }
+            }
+            else
+            {
+                exists = false;
+            }
 
 
 
-                //Create Pool
-                ApplicationPoolManager
-                    .Using(_Environment, _Log, _Server)
-                    .Create(settings.ApplicationPool);
+            //Create Pool
+            ApplicationPoolManager
+                .Using(_Environment, _Log, _Server)
+                .Create(settings.ApplicationPool);
 
 
 
-                //Site Settings
-                site = _Server.Sites.Add(
-                    settings.Name,
-                    settings.BindingProtocol.ToString().ToLower(),
-                    settings.BindingInformation,
-                    this.GetPhysicalDirectory(settings));
+            //Site Settings
+            site = _Server.Sites.Add(
+                settings.Name,
+                settings.BindingProtocol.ToString().ToLower(),
+                settings.BindingInformation,
+                this.GetPhysicalDirectory(settings));
 
-                if (settings.CertificateHash != null)
-                {
-                    site.Bindings[0].CertificateHash = settings.CertificateHash;
-                }
+            if (settings.CertificateHash != null)
+            {
+                site.Bindings[0].CertificateHash = settings.CertificateHash;
+            }
 
-                if (!String.IsNullOrEmpty(settings.CertificateStoreName))
-                {
-                    site.Bindings[0].CertificateStoreName = settings.CertificateStoreName;
-                }
+            if (!String.IsNullOrEmpty(settings.CertificateStoreName))
+            {
+                site.Bindings[0].CertificateStoreName = settings.CertificateStoreName;
+            }
 
-                site.ServerAutoStart = settings.ServerAutoStart;
-                site.ApplicationDefaults.ApplicationPoolName = settings.ApplicationPool.Name;
+            site.ServerAutoStart = settings.ServerAutoStart;
+            site.ApplicationDefaults.ApplicationPoolName = settings.ApplicationPool.Name;
 
 
 
-                //Security
-                this.SetAuthentication(settings);
-                this.SetAuthorization(settings);
+            //Security
+            this.SetAuthentication(settings);
+            this.SetAuthorization(settings);
 
-                return site;
-            }
+            return site;
+        }
+
+        protected const string webServerName = "webServer";
```

Comment:
What was wrong with declaring them inline, the variable name is longer than the word?